universe/node/api_space_attributes.go

```diff
@@ -0,0 +1,112 @@
+package node
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/momentum-xyz/ubercontroller/types/entry"
+	"github.com/momentum-xyz/ubercontroller/universe/api"
+	"github.com/pkg/errors"
+	"net/http"
+)
+
+func (n *Node) apiGetSpaceAttributes(c *gin.Context) {
+	spaceID, err := uuid.Parse(c.Param("spaceID"))
+	if err != nil {
+		err := errors.WithMessage(err, "Node: apiGetSpaceAttributes: failed to parse space id")
+		api.AbortRequest(c, http.StatusBadRequest, "invalid_space_id", err, n.log)
+		return
+	}
+
+	inQuery := struct {
+		PluginID string `form:"plugin_id" binding:"required"`
+		Name     string `form:"attribute_name" binding:"required"`
+	}{}
```

Comment:
please, move it to first line off the method
it's good practice to put all declarations first